contrib/kafka/kafkaeventsource-operator/build/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.6
+
+USER nobody
+
+ADD build/_output/bin/kafkaeventsource-operator /usr/local/bin/kafkaeventsource-operator
```

Comment:
Use `ENTRYPOINT ["/usr/local/bin/kafkaeventsource-operator"]` here instead of relying on `command` from the corresponding `.yaml` script.